components/coachindicator/index.css

```diff
@@ -0,0 +1,90 @@
+governing permissions and limitations under the License.
+*/
+
+@import "animation.css";
+
+.spectrum-CoachMarkIndicator {
+  position: relative;
+
+  margin: var(--spectrum-coachmark-indicator-gap);
+}
+
+.spectrum-CoachMarkIndicator-ring {
+  display: block;
+  position: absolute;
+
+  border-style: solid;
+  border-width: var(--spectrum-coachmark-indicator-ring-border-size);
+  border-radius: 50%;
+
+  &:nth-child(2) {
+    animation-delay: calc(var(--spectrum-coachmark-animation-indicator-ring-duration) * var(--spectrum-coachmark-animation-indicator-ring-center-delay-multiple));
+  }
+
+  &:nth-child(3) {
+    animation-delay: calc(var(--spectrum-coachmark-animation-indicator-ring-duration) * var(--spectrum-coachmark-animation-indicator-ring-outer-delay-multiple))
+  }
+}
+
+.spectrum-CoachMarkIndicator {
+  min-inline-size: calc(var(--spectrum-coachmark-indicator-ring-diameter) * 3);
+  min-block-size: calc(var(--spectrum-coachmark-indicator-ring-diameter) * 3);
+
+}
+
+.spectrum-CoachMarkIndicator-ring {
+  inset-block-start: calc(var(--spectrum-coachmark-indicator-ring-diameter) * 0.75);
+  inset-inline-start: calc(var(--spectrum-coachmark-indicator-ring-diameter) * 0.75);
+
+  inline-size: var(--spectrum-coachmark-indicator-ring-diameter);
+  block-size: var(--spectrum-coachmark-indicator-ring-diameter);
+  animation: pulse var(--spectrum-coachmark-animation-indicator-ring-duration) linear infinite;
+
+  &:nth-child(1) {
+    animation-delay: calc(var(--spectrum-coachmark-animation-indicator-ring-duration) * var(--spectrum-coachmark-animation-indicator-ring-inner-delay-multiple));
+  }
+}
+.spectrum-CoachMarkIndicator-ring {
+  border-color: var(--spectrum-coachmark-indicator-ring-default-color);
+}
+
+.spectrum-CoachMarkIndicator--light .spectrum-CoachMarkIndicator-ring {
+  border-color: var(--spectrum-coachmark-indicator-ring-low-contrast-color);
+}
+
+.spectrum-CoachMarkIndicator--dark .spectrum-CoachMarkIndicator-ring {
+  border-color: var(--spectrum-coachmark-indicator-ring-high-contrast-color);
+}
+
+.spectrum-CoachMarkIndicator--quiet {
+  min-inline-size: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 2.75);
+  min-block-size: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 2.75);
+
+  .spectrum-CoachMarkIndicator-ring {
+    inset-block-start: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 0.75);
+    inset-inline-start: calc(var(--spectrum-coachmark-quiet-indicator-ring-diameter) * 0.75);
+
+    inline-size: var(--spectrum-coachmark-quiet-indicator-ring-diameter);
+    block-size: var(--spectrum-coachmark-quiet-indicator-ring-diameter);
+    animation: pulse--quiet var(--spectrum-coachmark-animation-indicator-ring-duration) linear infinite;
+
+    &:nth-child(1) {
+      animation-delay: calc(var(--spectrum-coachmark-animation-indicator-ring-duration) * var(--spectrum-coachmark-quiet-animation-indicator-ring-inner-delay-multiple));
+    }
+  }
+}
```

Comment:
All of the quiet styles could probably be simplified to be custom property value overrides, rather than additional style rules, since they're all overwriting the values of the same styles set in the non-quiet version.